eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ExternalDependencyExposedCheck.java

```diff
@@ -204,7 +207,7 @@ private String getInvalidTypeNameFromTypeArgument(DetailAST typeArgumentToken) {
      * A helper function that checks for whether a class is from a valid internal dependency or is a suppression class
      *
      * @param typeName the type name of class
-     * @return true if the class  is a suppression class, otherwise, return false.
+     * @return true if the class is a suppression class, otherwise, return false.
```

Comment:
What is a "suppression class"?

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/7a297a16c188bdfa4c3e38884d6631b69fc87c92/eng/code-quality-reports/src/main/java/com/azure/tools/checkstyle/checks/ExternalDependencyExposedCheck.java#L24

Reply:
Thanks, Alan, Yes, any class's full package path that starts with any string in the VALID_DEPENDENCY_SET will be a suppression class